src/public-lib/components/SelectHelper.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from "react";
```

Comment:
this isn't really a component right? should probably put this into /checkedSelect/  and call it utils or something

Reply:
This is a shared utility class, for both `CheckedSelect` and `ChecklistSelect`, so maybe move this under `lib` instead of components?. `SelectUtils` might be a better name than `SelectHelper`.

Reply:
maybe both can be under `checklists` or something?

Reply:
It makes sense to put both these components and the utility file under the same folder. Maybe just keep the directory name `checkedSelect` for now? Checklist is a special kind of `CheckedSelect`. `CheckedSelect` should probably also be renamed to `DropdownCheckedSelect`, but I'll leave it as is for now.